comtypes/__init__.py

```diff
@@ -4,7 +4,16 @@
 import atexit
 from ctypes import *
 from ctypes import _SimpleCData
-from _ctypes import COMError
+try:
+    from _ctypes import COMError
+except ImportError as e:
+    msg = "\n".join((
+        "",
+        "COM technology not available (maybe it's the wrong platform).",
+        "Note that COM is only supported on Windows.",
+        "For more details, please check: "
+        "https://learn.microsoft.com/en-us/windows/win32/com."))
```

Comment:
The final `.` in this line is not necessary.
The dots in the URL and the dots as punctuation are confusing.